### PR TITLE
Position the modal over the navbar

### LINKED ISSUE #14269: Large modal is showing up under the top navbar
### Describe the bug

When a modal is open, it show be the most visible.

This is how it was
image

This is how it should be
image

### PR DESCRIPTION
**Pull request by MikeAlhayek**
Fix #14269

## After the PR Changes
image

## Before this PR
image

### CODE REVIEW

File: src/OrchardCore.Themes/TheAdmin/Assets/scss/admin.scss [hunk 1]
```diff
@@ -1,5 +1,10 @@
 @use 'sass:math';
 
+$zindex-sticky: 1020;
```
Comment (hishamco) [inline src/OrchardCore.Themes/TheAdmin/Assets/scss/admin.scss:3]: Where this variable has been used?

File: src/OrchardCore.Themes/TheAdmin/wwwroot/css/TheAdmin.css [hunk 2]
```diff
@@ -6308,7 +6309,7 @@ html[dir][data-theme=default] .toast-body {
 html[data-theme=default] .modal {
   position: fixed;
   top: 0;
-  z-index: 1055;
+  z-index: 1200;
```
Comment (hishamco) [inline src/OrchardCore.Themes/TheAdmin/wwwroot/css/TheAdmin.css:6312]: 1055 is greater than `$zindex-modal`?

Comment (MikeAlhayek) [reply src/OrchardCore.Themes/TheAdmin/wwwroot/css/TheAdmin.css:6312]: navbar is ($zindex-modal + 100) and the leftmenu is ($zindex-modal + 50). The modal need to be greater than ($zindex-modal + 100) so in this case `1150` is good to use. Check out the updated comments

File: src/OrchardCore.Themes/TheAdmin/Assets/scss/admin.scss (hunk 1 above)
Comment (MikeAlhayek) [reply src/OrchardCore.Themes/TheAdmin/Assets/scss/admin.scss]: This is a bootstrap variable. I moved it into a global scope to ensure that bootstrap honers it when BS scss files are compiled.

File: src/OrchardCore.Themes/TheAdmin/wwwroot/css/TheAdmin.css (hunk 2 above)
Comment (hishamco) [reply src/OrchardCore.Themes/TheAdmin/wwwroot/css/TheAdmin.css:6312]: I might need to test it, but I never mind If someone approves after a proper test

**Pull request merged**